oss-internship-2020/libuv/tests/test_array.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <linux/futex.h>
+#include <syscall.h>
+#include <uv.h>
+
+#include "gtest/gtest.h"
+#include "sandboxed_api/util/flag.h"
+#include "sandboxed_api/util/status_matchers.h"
+#include "uv_sapi.sapi.h"
+
+class UVTestArraySapiSandbox : public UVSandbox {
+ private:
+  std::unique_ptr<sandbox2::Policy> ModifyPolicy(
+      sandbox2::PolicyBuilder*) override {
+    return sandbox2::PolicyBuilder()
+        .AllowDynamicStartup()
+        .AllowExit()
+        .AllowFutexOp(FUTEX_WAKE_PRIVATE)
+        .AllowOpen()
+        .AllowSyscall(__NR_sysinfo)
+        .AllowWrite()
+        .BuildOrDie();
+  }
+};
+
+class UVTestArray : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    sandbox_ = std::make_unique<UVTestArraySapiSandbox>();
+    ASSERT_THAT(sandbox_->Init(), ::sapi::IsOk());
```

Comment:
Consistency: Either prefix with `::sapi` or plain `sapi`. Alternatively add a `using ::sapi::IsOk;` at the top and don't prefix at all.